azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/models/CallCompositeIncomingCallCancelledEvent.java

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui.calling.models;
+
+/**
+ * Incoming call cancelled event.
+ */
+public class CallCompositeIncomingCallCancelledEvent {
+    private final int code;
+    private final int subCode;
+    private final String callId;
+
+    /**
+     * Creates {@link CallCompositeIncomingCallCancelledEvent}.
+     * @param code call end code.
+     * @param subCode call end sub code.
+     * @param callId call id.
+     */
+    public CallCompositeIncomingCallCancelledEvent(final int code,
```

Comment:
Does it need to be public?

Reply:
All constructors are public.

Reply:
Not all, only those which we intend Contoso to instantiate. For instance, CallCompositeDebugInfo, CallCompositeCallHistoryRecord